Dockerfile

```diff
@@ -2,11 +2,17 @@
 FROM python:3.9
 
 # Set working directory.
+WORKDIR /the/workdir/path
```

Comment:
I think you want to set this to `/code`, based on the commands you've added below.